### ISSUE TITLE
Laminas Cli Command Candidate?

### ISSUE THREAD

**Issue by tyrsson** [PacktPublishing/Laminas-Web-Development#31]
https://github.com/PacktPublishing/Laminas-Web-Development/blob/fa5b6dcfce329517d01774b0bcc8f75d42474539/chapter10/whatstore/bin/load_database.php#L9

Curious as to why this was not done with Laminas-Cli.

Benefits:
Could have been interactive.
Would have provided an example of how to provide command line tooling via laminas cli.
Since Laminas Cli commands can have factories it would have simplified the workflow since the factory would have access to the service container, and config service.

**Comment by fgsl** [comment]
Yes, you are right. It could be an opportunity to talk about laminas-cli. But I don't know if the database population script is appropriated to explore the interactive aspect of laminas-cli. In this case, every function will be executed. There are no alternatives. Then I think that is a limited example.

I already have talked about the main dependency for laminas-cli, but I wrote a full chapter about it, with several examples. For this case, it doesn't seem essential to me, considering the limited context. 

But I will reflect about it. I will consider other changes. Thank you very much! :)

![image](https://user-images.githubusercontent.com/128915/187081131-377c0048-f9d0-461b-8ffe-2bf2fc1a66ff.png)

**Comment by tyrsson** [comment]
True enough. I recently created an installer for the application that I was working on. The command excepted the db connection details, I used that in conjunction with Laminas\Config to write the database connection info for the application while the db was being populated so to remove the need for a web installer which would then need to be removed or cause a security issue. Since any build for the application on any given server will require shell access to run composer it only seemed a logical way to handle it, for me at least.
